torch/csrc/jit/tensorexpr/stmt.h

```diff
@@ -78,16 +90,37 @@ class Block : public StmtNode<Block> {
     }
     return new Block(valid_stmts);
   }
+
   int nstmts() const {
     return stmts_.size();
   }
-  Stmt* stmt(int index) const {
-    return stmts_[index];
+
+  void append_stmt(Stmt *s) {
+    stmts_.push_back(s);
+    set_parent(s, this);
+  }
+  bool replace_stmt(Stmt* old_stmt, Stmt* new_stmt) {
```

Comment:
Minor: many system provides another version to insert "std::vector<Stmt*>". That way, we avoid introducing too many unnecessary level of Blocks.